src/pick.c

```diff
@@ -446,6 +448,14 @@ selected_choice(void)
 			while (cursor_position < query_length
 			    && isu8cont(query[++cursor_position]));
 			break;
+		case PAGE_DOWN:
+			if (yscroll + lines - 1 < choices_count)
```

Comment:
Good point! I just changed the page up behavior to always scroll even
when less than a page has been scrolled, in this scenario the vertical
scroll is removed and the first choice is selected. For page down, we
could either:

- Scroll and only show the last choice

- Select the last choice but do not scroll any further. This is somewhat
  more consistent to the current page up behavior and how mutt(1)
  behaves as well.

I vote for the later, what's your take?